{"uuid":"8dd559f1-be0b-4f26-8b90-7e94a80b890c","historyId":"697be5c623328828f28b2546da814505","testCaseId":"[engine:junit-jupiter]/[class:com.Rently.Business.Integration.UsuarioServiceImplIntegrationTest]/[method:create_InvalidPhone_ShouldThrowException()]","testCaseName":"CREATE - Teléfono inválido debe lanzar excepción","fullName":"com.Rently.Business.Integration.UsuarioServiceImplIntegrationTest.create_InvalidPhone_ShouldThrowException","labels":[{"name":"junit.platform.uniqueid","value":"[engine:junit-jupiter]/[class:com.Rently.Business.Integration.UsuarioServiceImplIntegrationTest]/[method:create_InvalidPhone_ShouldThrowException()]"},{"name":"host","value":"Laumcs"},{"name":"thread","value":"24472@Laumcs.main(1)"},{"name":"framework","value":"junit-platform"},{"name":"language","value":"java"},{"name":"package","value":"com.Rently.Business.Integration.UsuarioServiceImplIntegrationTest"},{"name":"testClass","value":"com.Rently.Business.Integration.UsuarioServiceImplIntegrationTest"},{"name":"testMethod","value":"create_InvalidPhone_ShouldThrowException"},{"name":"suite","value":"UsuarioService - Integration Tests"}],"links":[],"name":"CREATE - Teléfono inválido debe lanzar excepción","status":"passed","stage":"finished","description":"","steps":[],"attachments":[],"parameters":[],"start":1759764772389,"stop":1759764772819}